src/app/test_executive/hard_fork.ml

```diff
@@ -17,6 +18,75 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
 
   type dsl = Dsl.t
 
+  module Balance = struct
+    open Currency
```

Comment:
That pulls in quite a bit. Is it just the `Balance` module you need?

Reply:
Yes, just `Balance` as far as I can tell. How about the following?

```suggestion
    module Balance = Currency.Balance
```